pages/new-tab/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { createRoot } from 'react-dom/client';
 import '@src/index.css';
-import '@extension/ui/lib/global.css';
+import '../dist/global-out.css';
```

Comment:
Maybe your plugin are able to add this import during build to `.js` bundled file?
We can avoid those imports in each file.

Reply:
huh that's a great idea, perhaps will solve my build time error not found this file during build